addons/l10n_ch/__manifest__.py

```diff
@@ -9,28 +9,21 @@
     'description': """
 Swiss localization
 ==================
-This module defines a chart of account for Switzerland (Swiss PME/KMU 2015), taxes and enables the generation of ISR and QR-bill when you print an invoice or send it by mail.
-
-An ISR will be generated if you specify the information it needs :
-    - The bank account you expect to be paid on must be set, and have a valid postal reference.
-    - Your invoice must have been set assigned a bank account to receive its payment
-      (this can be done manually, but a default value is automatically set if you have defined a bank account).
-    - You must have set the postal references of your bank.
-    - Your invoice must be in EUR or CHF (as ISRs do not accept other currencies)
+This module defines a chart of account for Switzerland (Swiss PME/KMU 2015), taxes and enables the generation of QR-bill when you print an invoice or send it by mail.
 
 A QR-bill will be generated if:
     - The partner set on your invoice has a complete address (street, city, postal code and country) in Switzerland
     - The option to generate the Swiss QR-code is selected on the invoice (done by default)
     - A correct account number/QR IBAN is set on your bank journal
     - (when using a QR-IBAN): the payment reference of the invoice is a QR-reference
 
-The generation of the ISR and QR-bill is automatic if you meet the previous criteria.
+The generation of the QR-bill is automatic if you meet the previous criteria.
 
 Here is how it works:
-    - Printing the invoice will trigger the download of three files: the invoice, its ISR and its QR-bill
-    - Clicking the 'Send by mail' button will attach three files to your draft mail : the invoice, the ISR and the QR-bill.
+    - Printing the invoice will trigger the download of two files: the invoice and its QR-bill
+    - Clicking the 'Send by mail' button will attach two files to your draft mail : the invoice and the QR-bill.
     """,
-    'version': '11.0',
+    'version': '16.0',
```

Comment:
Why changing that ? And why increasing it by 5 ? It's the module version, not Odoo's version ^^ If you have doubts about its use, come ask me, I can explain :)